ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -428,13 +428,7 @@ public void run() {
               try {
                 StreamedRow row = queryStream.next();
                 terminal.printStreamedRow(row);
-                if (row.getErrorMessage() != null) {
-                  // got an error in the stream, which means we have reached the end.
-                  // the stream interface that queryStream uses isn't smart enough to figure
-                  // out when the socket is closed, so just break here since we know there will
-                  // be nothing more to read.
-                  LOGGER.debug("Going to stop reading results for row {} since there was an error"
```

Comment:
This was the log spam, right?

Reply:
No, the log spam was lots of ERROR and WARNING lines, both from the Streams lib and KSQL code, because an exception was thrown by a processor node.